Assets/Scripts/Scoreboard/ScoreboardManager.cs

```diff
@@ -140,8 +141,9 @@ private IEnumerator NextCrime()
         }
         else
         {
+            DisplayProgress?.Invoke();
             // Start next round
-            yield return new WaitForSeconds(newCrimeDelay);
+            yield return new WaitForSeconds(5f);
```

Comment:
This has to match up with the timing in the Scoreboard class, right? Seems easily breakable but will work fine for the time being. Perhaps you could expose the time values in the scoreboards and use them here?

Reply:
I have now exposed the value and made individual scoreboards adjust their timings based on this value 👍